lib/utils/util.dart

```diff
@@ -23,6 +23,7 @@ class Util {
   static const String empty = '';
   static const String countryIso = 'countryIso';
 
+  static const String preferredChurchModalShown = 'preferredChurchModalShown';
```

Comment:
My preference would be to not have this here but in the impactgroupsrepo. And then to rename the field to have Key as a suffix.